src/main/java/com/vaadin/demo/component/grid/GridCellFocus.java

```diff
@@ -0,0 +1,67 @@
+package com.vaadin.demo.component.grid;
+
+import com.vaadin.demo.DemoExporter; // hidden-source-line
+import com.vaadin.demo.domain.DataService;
+import com.vaadin.demo.domain.Person;
+import com.vaadin.flow.component.grid.CellFocusEvent;
+import com.vaadin.flow.component.grid.Grid;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.textfield.TextArea;
+import com.vaadin.flow.router.Route;
+
+import java.util.List;
+
+@Route("grid-cell-focus")
+public class GridCellFocus extends Div {
+
+    public GridCellFocus() {
+        Grid<Person> grid = new Grid<>(Person.class, false);
+        grid.addColumn(Person::getFirstName)
+                .setKey("firstName")
+                .setHeader("First name");
+        grid.addColumn(Person::getLastName)
+                .setKey("lastName")
+                .setHeader("Last name");
+        grid.addColumn(Person::getEmail)
+                .setKey("email")
+                .setHeader("Email");
+        grid.addColumn(Person::getProfession)
+                .setKey("profession")
+                .setHeader("Profession");
+
+        List<Person> people = DataService.getPeople();
+        grid.setItems(people);
+
+        TextArea textArea = new TextArea();
```

Comment:
Same thing here about making it `readonly`.
```suggestion
        TextArea textArea = new TextArea();
        textArea.setReadonly(true);
```

Reply:
Fixed.